about_solid.html

```diff
@@ -84,13 +84,13 @@ <h3>Fully interoperable standards</h3>
                     </div>
                     <div class="col-md-12">
                         <h3>Access</h3>
-                        <p>With Solid's Authentication and Authorization systems, you determine which people and
+                        <p>With Solid's Authentication and Authorization systems, one can determine which people and
                             applications can access your data.
-                            You grant or revoke access to any slice of your data as needed. Consequently, you can do
-                            more with your data, because the
-                            applications you decide to use can be granted access to a wider and more diverse set of
+                            People can grant or revoke access to any slice of their data as needed. Consequently, people can do
+                            more with their data, because the
+                            applications they decide to use can be granted access to a wider and more diverse set of
```

Comment:
```suggestion
                            Entities can grant or revoke access to any slice of their data as needed. Consequently, entities can do
                            more with their data, because the
                            applications they decide to use can be granted access to a wider and more diverse set of
```